SECURITY.md

```diff
@@ -22,11 +22,41 @@ If it is not triaged in a couple of days, feel free to tag `@the-frey` or `@jake
 
 ## Critical bug or security issue 💥
 
-For a critical security bug, please do not publicly disclose it, or post on GitHub. Instead, go to [Discord](https://discord.gg/wHdzjS5vXx), and alert the core engineers:
+If you're here because you're trying to figure out how to notify us of a security issue, 
+
+go to [Discord](https://discord.gg/wHdzjS5vXx), and alert the core engineers:
```

Comment:
```suggestion
If you're here because you're trying to figure out how to notify us of a security issue, go to [Discord](https://discord.gg/wHdzjS5vXx), and alert the core engineers:
```

Reply:
Minor formatting.